### PR TITLE
movie: Pitch Black (2000) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Token action film. Rating 2/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -18,6 +18,59 @@ https://en.wikipedia.org/wiki/List_of_science_fiction_films_of_the_2000s
...
+desert planet [A main novelty of the film is the baked in perpetual daylight by three suns desert planet on which it is set.]
+human vs. nature [The surviving crew and passengers of the Hunter-Gratzner spaceship struggled to survive on the desert planet on which they crash-landed.]
+trust in a potential foe [Carolyn and Johns took a calculated risk by releasing the dangerous criminal Riddick and accepting him into their party. They remained suspicious and mistrustful of him throughout the course of the film, but needed his help to survive on the planet.]
+human vs. monster [Central to the story was the surviving crew and passengers of the Hunter-Gratzner spaceship trying to not get killed by the man-eating creatures that inhabited the desert planet on which they'd crash-landed.]
+monster [The desert planet was inhabited by grotesque, man-eating creatures.]
+human character stereotype [The hero of the story Riddick was a stereotypical hard-boiled, lone wolf, in-it-for-only-himself Hollywood antihero figure.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:56]: Not keen on using this when someone "feels stereotypical" but we can't put the finger on it enough to invent a theme name. My suggestion is either take it out or invent a theme-worthy name and put it as a template parameter (or write a theme def).

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 2]
```diff
@@ -18,6 +18,59 @@ https://en.wikipedia.org/wiki/List_of_science_fiction_films_of_the_2000s
...
+speculative ability [A main novelty of the story was that Riddick had surgically altered eyes that allowed him to see in the dark.]
+
+:: Minor Themes
+speculative spaceship [The futuristic interstellar vessel Hunter-Gratzner was transporting 40 passengers in cryostasis.]
+cryogenic technology [The Hunter-Gratzner spaceship was transporting 40 cryo-sleeping passengers.]
+habitable planet in a trinary star system [Three suns featured prominently in the desert planet sky.]
+organized religion [The Muslim preacher Imam and his three young acolytes were traveling to a place called New Mecca.]
+role models [The tomboy Jack immediately took a liking to the hardened criminal Riddick and soon thereafter began to emulate him.]
+fear [The marooned Hunter-Gratzner crew and passengers pointedly discussed their fears, or lack thereof, of the vicious creatures that went flying around the desert planet at night, killing anything in their path.]
+gender bending [The young girl Jack was posing as a boy to avoid attention.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:67]: Was there more to it or do we really want to use this theme when someone just disguised themselves as the opposite sex?

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 3]
```diff
@@ -18,6 +18,59 @@ https://en.wikipedia.org/wiki/List_of_science_fiction_films_of_the_2000s
...
+personal practical dilemma [Riddick forced Carolyn to choose between leaving her fellow marooned travelers for dead on the planet and escaping with him, and staying behind on the planet to die together with them.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:68]: I compare with https://www.themeontology.org/theme.php?name=sacrificing%20a%20friend%20to%20save%20one%27s%20self and think we might do well to introduce "abandoning someone to save one's self"..

**Review by odinlake — COMMENTED** [review]
Comments inline

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:56]: Okay, let's just delete it.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:67]: I see there's a precedent for using it this way in [The Two Gentlemen of Verona (1593)](https://www.themeontology.org/story.php?name=play%3A%20The%20Two%20Gentlemen%20of%20Verona%20(1593)). So why not?

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 3 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt]: I added a stub entry. Will define in another pull request.

**Comment by odinlake** [comment]
That’s true and I should perhaps have motivate it better.

Like many Shakespeare stories, this one’s charged with sexuality and gender
issues and one need not look far to find people reading gender bending into
Julia’s character:

http://www.inquiriesjournal.com/articles/590/the-two-gentlemen-of-verona-a-satiric-perspective-on-courtly-love

Can you say something of that sort for your case?

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt]: Let's see. Unfortunately, it appears that scholars have not subjected **Pitch Black** to the same level of scrutiny as they have **The Two Gentlemen of Verona** - a travesty, I know.

But I did update the motivate to include that Jack had taken on a male persona in addition to dressing in men's clothing.

**Comment by odinlake** [comment]
> Let's see. Unfortunately, it appears that scholars have not subjected 
> *Pitch Black* to the same level of scrutiny as they have *The Two 
> Gentlemen of Verona* - a travesty, I know.

Palpable sarcasm, but misdirected. The fact that someone else has saved 
us time by already motivating the theme in the Shakespeare story is 
neither here nor there.

I still don't know if there was a *social issue* here. If all there was 
to it is the practical motive of remaining under cover then it would 
seem not. Now, if someone does it because they enjoy it, for peer 
pressure, or for public entertainment or something then it becomes a lot 
more like the themes homosexuality, transsexuality, open sexuality etc. 
which we have decided are inherently about *sexual norms*. Was someone 
at least shocked when she/he was revealed as a girl? That'd be a better 
motivation.

Anyway, whatever. Just imagine you had to defend this in a workshop full 
of Barry Smith's and literary critics.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt]: Motivation updated accordingly.

**Pull request merged**